docs/Examples/BinaryRequest/Main.purs

```diff
@@ -2,14 +2,14 @@ module Examples.BinaryRequest.Main where
 
 import Prelude
 import Effect.Console (log)
+import HTTPure (Request, ResponseM, ServerM, ok, serve, toBuffer)
 import Node.Buffer (Buffer)
-import HTTPure (Request, ResponseM, ServerM, toBuffer, serve, ok)
 
 foreign import sha256sum :: Buffer -> String
 
 -- | Respond with file's sha256sum
 router :: Request -> ResponseM
-router { body } = toBuffer body >>= sha256sum >>> ok
+router request = toBuffer request.body >>= sha256sum >>> ok
```

Comment:
I think you can revert this whole file now

Reply:
Reverted